src/nunit-gui/Views/MainForm.Designer.cs

```diff
@@ -907,21 +898,21 @@ private void MainForm_DragEnter(object sender, DragEventArgs e)
         private TestPropertiesView propertiesView;
         private System.Windows.Forms.TabPage tabPage2;
         private XmlView xmlView;
-        private ToolStripSeparator toolStripSeparator1;
-        private ToolStripMenuItem selectRuntimeToolStripMenuItem;
-        private ToolStripMenuItem defaultRuntimeToolStripMenuItem;
-        private ToolStripMenuItem processModelToolStripMenuItem1;
-        private ToolStripMenuItem defaultProcessToolStripMenuItem;
-        private ToolStripMenuItem inProcessToolStripMenuItem;
-        private ToolStripMenuItem singleProcessToolStripMenuItem;
-        private ToolStripMenuItem multipleProcessToolStripMenuItem;
-        private ToolStripSeparator toolStripSeparator2;
-        private ToolStripMenuItem loadAsX86ToolStripMenuItem;
-        private ToolStripMenuItem domainUsageToolStripMenuItem1;
-        private ToolStripMenuItem defaultDomainToolStripMenuItem;
-        private ToolStripMenuItem singleDomainToolStripMenuItem;
-        private ToolStripMenuItem multipleDomainToolStripMenuItem;
-        private ToolStripMenuItem notYetImplementedToolStripMenuItem;
+        private System.Windows.Forms.ToolStripSeparator toolStripSeparator1;
+        private System.Windows.Forms.ToolStripMenuItem selectRuntimeToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem defaultRuntimeToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem processModelToolStripMenuItem1;
+        private System.Windows.Forms.ToolStripMenuItem defaultProcessToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem inProcessToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem singleProcessToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem multipleProcessToolStripMenuItem;
+        private System.Windows.Forms.ToolStripSeparator toolStripSeparator2;
+        private System.Windows.Forms.ToolStripMenuItem loadAsX86ToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem domainUsageToolStripMenuItem1;
+        private System.Windows.Forms.ToolStripMenuItem defaultDomainToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem singleDomainToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem multipleDomainToolStripMenuItem;
+        private System.Windows.Forms.ToolStripMenuItem notYetImplementedToolStripMenuItem;
```

Comment:
in the commit that was added a while ago that added support for Drag and drop the user added code directly to the auto-generated designer file. When they did they also added a using to the top of the file. Later in one of your commits you change that form and so it was generated again. It dropped the event (which is why it stopped working), so this was a move back to "default" generated code. Winforms usually has no usings in the designer file so I took out the using and added back in the fully qualified name.